axelor-base/src/main/java/com/axelor/apps/base/service/imports/ImportCityServiceImpl.java

```diff
@@ -153,4 +180,145 @@ private void deleteTempFiles(File configXmlFile, File dataCsvFile) {
       e.printStackTrace();
     }
   }
+
+  /**
+   * extracting file from the zip
+   *
+   * @param downloadFileName : zip fileName to download from internet
+   * @return
+   */
+  @Override
+  public void downloadZip(String downloadFileName) throws IOException {
+    File downloadFile;
+
+    File cityTextFile;
+    ZipFile zipFile;
+
+    URL url;
+
+    Path path =
+        Paths.get(
+            AppSettings.get()
+                .get(
+                    "file.upload.dir",
+                    "{java.io.tmpdir}"
+                        + File.separator
+                        + "axelor"
+                        + File.separator
+                        + "attachments"));
+
+    downloadFile = new File(path + File.separator + downloadFileName);
```

Comment:
check if you can do it like
new File(path.toString(), downloadFileName);
so we don't need to access separator here